util/dockerfiles/1.13.1/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:testing
```

Comment:
@ben-albrecht This `Dockerfile` is tied to "testing", a rolling Debian release. Should we revise this to a fixed release? Currently, "jessie" = "stable" and "stretch" = "testing".